yarn-project/circuits.js/package.json

```diff
@@ -35,7 +35,7 @@
     "clean": "rm -rf ./dest .tsbuildinfo",
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T eslint --fix ./src && run -T prettier -w ./src",
-    "remake-constants": "node --loader ts-node/esm src/scripts/constants.in.ts && prettier -w src/constants.gen.ts && cd ../../l1-contracts && ../foundry/bin/forge fmt",
+    "remake-constants": "node --loader ts-node/esm src/scripts/constants.in.ts && prettier -w src/constants.gen.ts && cd ../../l1-contracts && forge fmt",
```

Comment:
Maybe this wasn't meant to be included in the commit?

Reply:
It was: there is no `aztec-packages/foundry` anymore. In fact, I can't see how/where `remake-constants` is called anywhere anymore other than manually.